ops/lib.rs

```diff
@@ -169,19 +165,42 @@ fn codegen_v8_sync(core: &TokenStream2, f: &syn::ItemFn) -> TokenStream2 {
     let state = unsafe { &*(state_refcell_raw as *const std::cell::RefCell<#core::OpState>) };
 
     let op_state = &mut state.borrow_mut();
-    let result = Self::call::<#type_params>(op_state, a, b);
+    let result = Self::call::<#type_params>(op_state, #args_tail);
 
     op_state.tracker.track_sync(op_id);
 
     #ret
   }
 }
 
+fn codegen_args(
+  core: &TokenStream2,
+  f: &syn::ItemFn,
+  rust_i0: usize,
+  v8_i0: usize,
+) -> (TokenStream2, TokenStream2) {
```

Comment:
Can you explain (maybe in a comment) what rust_i0 and v8_i0 do?

Add `assert!(rust_i0 <= v8_i0);` ?

Reply:
> Add `assert!(rust_i0 <= v8_i0);` ?

Nope, `rust_i0 <= v8_i0` won't always be true and likely won't with an upcoming change (depending on the impl path, since there are 2 alternatives).

`rust_i0` and `v8_i0` represent the starting indexes of generic arguments in rust & v8/js respectively